early-bird/src/components/edit-profile-page/EditProfilePage.js

```diff
@@ -0,0 +1,210 @@
+import React from "react";
+import "./EditProfilePage.scss";
+import editLogo from "../../illustrations/Saly-edit-profile-page.svg";
+import TextField from "@material-ui/core/TextField";
+import { Update } from "../../services/UsersService.js";
+import { GetUserId } from "../../services/AccountService";
+import { useState, useEffect } from "react";
+import { GetUserById } from "../../services/UsersService";
+import Snackbar from "@material-ui/core/Snackbar";
+import MuiAlert from '@material-ui/lab/Alert';
+
+function EditProfilePage() {
+  const [firstName, setFirstName] = useState(" ");
+  const [lastName, setLastName] = useState(" ");
+  const [password, setPassword] = useState(null);
+  const [confirmPassword, setConfirmPassword] = useState(null);
+
+  const [loading, setLoading] = useState(true);
+  const [openSnack, setOpenSnack] = React.useState(false);
+
+  const [errorFlags, setErrorFlag] = useState([false, false, false, false]);
+  let errorFlagsAux = [...errorFlags];
+
+  function Alert(props) {
```

Comment:
change props to something else, it's confusing and if you will want to introduce component props there will be some weird errors